### PR TITLE
fscrypt: add page

### PR DESCRIPTION
**Pull request by ezr**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):** fscrypt version v0.3.0

### CODE REVIEW

**Comment by ezr** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/1298720895) for this PR failed with the following error(s):
> 
> ```
> pages/linux/fscrypt.md:
> Error: Parse error on line 4:
> ....com/google/fscrypt>- Prepare the root
> -----------------------^
> Expecting 'END_INFORMATION_LINK_URL', got 'NEWLINE'
> pages/linux/fscrypt.md:4: TLDR011 Page never contains more than a single empty line
> ```
> 
> Please fix the error(s) and push again.

I was missing a period at the end of line 4. I added the period with a second commit. I don't know how to push again.

File: pages/linux/fscrypt.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Go tool for managing Linux filesystem encryption.
+> More information: <https://github.com/google/fscrypt>.
+
+- Prepare the root filesystem for use with fscrypt:
+
+`fscrypt setup`
+
+- Enable filesystem encryption for a directory:
+
+`fscrypt encrypt /path/to/dir`
```
Comment (CleanMachine1) [inline pages/linux/fscrypt.md:12]:
```suggestion
`fscrypt encrypt {{path/to/directory}}`
```
Following our syntax.

File: pages/linux/fscrypt.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Unlock an encrypted directory:
+
+`fscrypt unlock /path/to/dir`
```
Comment (CleanMachine1) [inline pages/linux/fscrypt.md:16]:
```suggestion
`fscrypt unlock {{path/to/encrypted_directory}}`
```

File: pages/linux/fscrypt.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Lock an encrypted directory:
+
+`fscrypt lock /path/to/dir`
```
Comment (CleanMachine1) [inline pages/linux/fscrypt.md:20]:
```suggestion
`fscrypt lock {{path/to/encrypted_directory}}`
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @ezr

**Pull request merged**